api/views.py

```diff
@@ -87,6 +91,21 @@ def get_queryset(self):
             queryset = queryset.order_by('-services__cost_service')
         return queryset
 
+    @action(['get', 'post'], detail=False)

```

Comment:
Я точно не уверен, но у меня ошибка падала в каком-то проекте, если писать lowercase, а потом проверку капсом делать. Если все работает, то замечательно
```suggestion
    @action(['GET', 'POST'], detail=False)
```